toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -426,15 +428,6 @@ func customizeOSContents(ic *ImageCustomizerParameters) error {
 		ic.config.OS = &imagecustomizerapi.OS{}
 	}
 
-	// Check if dm-verity is enabled on the base image.
-	verityEnabled, err := isDmVerityEnabled(ic.rawImageFile)
-	if err != nil {
-		return err
-	}
-	if verityEnabled && !ic.config.CustomizePartitions() {
-		return fmt.Errorf("dm-verity is enabled on the base image so partitions must be specified")
```

Comment:
This logic should have been removed? I think your last commit about leveraging pre-configured partition for verity can support to remove this limitation already?

Reply:
No, in the other feature, the verity partitions are pre-created but verity is not actually initialized. Hence, the image isn't "verity enabled".